openhtf/core/test_state.py

```diff
@@ -68,6 +68,26 @@ class DuplicateAttachmentError(Exception):
   """Raised when two attachments are attached with the same name."""
 
 
+class Subscribable(object):
+  """Subscribable object."""
```

Comment:
Man, you have such a high bar!

Yeah, I left this as a placeholder then forgot about it, whoops!